### ISSUE TITLE
“MacDown” cannot be opened because the developer cannot be verified.

### ISSUE THREAD

**Issue by xploreout** [MacDownApp/macdown#1249]
Above message appears when trying to start the app after `brew install --cask macdown`.  Saw similar error in past history but thought it was resolved.

**Comment by worikgh** [comment]
Me too

**Comment by orafaaraujo** [comment]
I'm having the same :( 

Any news on it?

**Comment by janbrasna** [comment]
https://houtianze.github.io/brew/mac/2020/02/18/brew-no-quarantine.html

<blockquote><div><strong><a href="https://houtianze.github.io/brew/mac/2020/02/18/brew-no-quarantine.html">The `brew install` parameter `--no-quarantine`</a></strong></div><div>Tired of seeing the Mac Security warning dialog box each time after you installed some Awesome.app using brew? There is a (semi-)hidden parameter to fix that...</div></blockquote>

**Comment by janbrasna** [comment]
Actually it's been mentioned here as well: https://github.com/MacDownApp/macdown/issues/1106#issuecomment-561853451

**Comment by chrisjf** [comment]
This is also an issue when downloading the app from the Github releases page, as well as the developer's website (not just when installing using Homebrew). 

I believe this is a app notarization issue. More info here on how to properly resolve this issue: https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution See the section "Notarize Your App Automatically as Part of the Distribution Process" 🙂 

<blockquote><div><strong><a href="https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution">Apple Developer Documentation</a></strong></div></blockquote>

**Comment by janbrasna** [comment]
@chrisjf You're right. See the above mentioned thread for reasons the author is not willing to join the Developer program, and a followup #1127 what to do next.

**Comment by fnc12** [comment]
Why don't you post in in App Store?

**Comment by sweetppro** [comment]
> Why don't you post in in App Store?

That cost $100 per year

**Comment by fnc12** [comment]
Can I help you with publishing it on App Store? I have a permanent [developer account](https://apps.apple.com/us/developer/евгений-захаров/id644182711?l=ru). I can post this app with a disclaimer that it is developed by community not by me. App Store is a very comfortable way of installing software. It is much more popular than Microsoft Store.

**Comment by fnc12** [comment]
Yes no? What is the answer?

**Comment by janbrasna** [comment]
@fnc12 The options of publishing under different Developer IDs have been discussed before: https://github.com/MacDownApp/macdown/issues/1106#issuecomment-557385038

**Comment by tir38** [comment]
Owner was clear why they haven't fixed this here: https://github.com/MacDownApp/macdown/issues/1106#issuecomment-555866606

Def a bummer, but it's their decision 🤷

**Comment by lsloan** [comment]
There are other solutions.  Workarounds, actually…

* Add instructions to the README explaining…
  1. How to open and approve the app from the Finder with right-click on the MacDown icon.
  2. How to install with Homebrew using the [`--no-quarantine`](https://houtianze.github.io/brew/mac/2020/02/18/brew-no-quarantine.html) flag.
  3. How to use `sudo xattr -r -d com.apple.quarantine ~/../../Applications/MacDown.app` to remove the quarantine from the app whether it's installed with either of the above two methods.
* Revisit whether quarantine can be disabled within the Homebrew formula.  It appears that it was last checked in 2019.  It's three or more years since then.  Maybe Homebrew can bypass quarantine without the user specifying the option.
